### PR TITLE
Add landing message. #1

### PR DESCRIPTION
**Pull request by amirhamzan-harry**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,9 +3,9 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Welcome to potionfy</title>
 </head>
 <body>
-    <p>Welcome to potionfy</p>
+    <p>Welcome to potionfy! Enjoy your stay!!</p>
```
Comment (amirhamzan-hermione) [inline index.html]: Could you remove 'Enjoy your stay!' string and make the string p tag to red?

**Review by amirhamzan-harry — COMMENTED** [review]
Hey do check the comment part

**Review by amirhamzan-hermione — COMMENTED** [review]
Hermione did this

File: index.html (hunk 1 above)
Comment (amirhamzan-harry) [reply index.html]: Sure! Can do

**Review by amirhamzan-hermione — CHANGES_REQUESTED** [review]
Harry agree to do this

**Review by amirhamzan-hermione — APPROVED** [review]
Good job buddy!

**Pull request merged**